src/app/plugins/connectfour.plugin.ts

```diff
@@ -189,8 +184,8 @@ class ConnectFourGame {
     // We use a very simple evaluation: -(longest player one chain)
     // This causes Lion to simply try to minimize Player 1's success,
     // While our _checkWin case takes over when it's time to clinch victory.
-    if (depth === this._aiDepth) {
-      return this._longestChainOnBoard(1) - this._longestChainOnBoard(-1);
+    if (depth === 1) {
```

Comment:
Magic number, i know there was a comment about this, but it would be nice to have it stored as a field

Reply:
That depth actually is wrong, thanks for pointing that out. There is a field, it's this._aiDepth. I forgot to add it back.